dimagi/utils/couch/undo.py

```diff
@@ -32,3 +35,13 @@ def soft_delete(self, domain_included=True):
             record.save()
             self.save()
             return record
+
+
+def is_deleted(doc):
+    """
+    Guess if a document is deleted. Returns False if we're not sure.
```

Comment:
If False means "not sure" rather than "no" should this be `maybe_deleted`? although functions marked "maybe" aren't great...`has_doc_type_deleted`?